app/jobs/attach_files_to_work_job.rb

```diff
@@ -23,22 +23,12 @@ def perform(work, uploaded_files)
     # @param [Hyrax::Actors::FileSetActor] actor
     # @param [UploadedFileUploader] file
     def attach_content(actor, file)
-      case file.file
-      when CarrierWave::SanitizedFile
+      if file.file.is_a? CarrierWave::SanitizedFile
         actor.create_content(file.file.to_file)
-      when CarrierWave::Storage::Fog::File
-        import_url(actor, file)
+      elsif file.url.present?
```

Comment:
No.  I was confused by that at first also.  

`file` is of our `Hyrax::UploadedFileUploader` class, and that inherits from `CarrierWave::Uploader::Base`, so `file.respond_to?(:url)` is always true.  This is easy to demonstrate:
```ruby
> z=CarrierWave::Uploader::Base.new
 => #<CarrierWave::Uploader::Base:0x007f9de4783028 @model=nil, @mounted_as=nil> 
> z.respond_to?(:url)
 => true 
```

That is not to be confused with `file.file`, of course, which may be different things.